src/current/cockroachcloud/provision-a-cluster-with-terraform.md

```diff
@@ -141,6 +141,8 @@ In this tutorial, you will create a CockroachDB {{ site.data.products.standard }
       - Under `usage_limits`, set `provisioned_virtual_cpus` to the required maximum vCPUs for the cluster.
       - Under `regions`, add the names of one or more regions for the cluster.
       - To optionally enable [deletion protection]({% link cockroachcloud/basic-cluster-management.md %}#enable-deletion-protection), set `delete_protection` to `true`.
+
```

Comment:
i think (?) this would be where i'd put the anchor link to the "one change policy (tm)" if the anchor thing is even possible, maybe it's not (see prev comment)

so like 

```
<a name="pweez-one-change"></a>
```

Reply:
I decided to link to the general Retention section. But thanks for pointing out the earlier subtle language!